### PR TITLE
[WIP] GUI2 Help Browser

### PR DESCRIPTION
**Pull request by babaissarkar**
Screenshot from 2024-07-29 18-16-03

### CODE REVIEW

**Comment by Wedge009** [comment]
Oh, good on you for having a go at this!

**Comment by CelticMinstrel** [comment]
Is this now disentangled from the branch you're requesting it be merged into? I don't see any of the commits from that branch, which seems to imply this was in fact rebased on top of it. Is that accurate? Or are those first three commits still mixed in with the work from that branch?

Either way it'll need to be rebased (or maybe redone a bit) once I actually fix up that branch. I'll let you know when that is.

**Comment by babaissarkar** [comment]
> Either way it'll need to be rebased (or maybe redone a bit) once I actually fix up that branch. I'll let you know when that is.

Yes, this one still needs to be rebased, and there are those mixed commits that need to be cleaned up.

File: src/gui/widgets/rich_label.cpp [hunk 1]
```diff
@@ -207,24 +206,24 @@ void rich_label::add_image(config& curr_item, std::string name, std::string alig
...
-	DBG_GUI_RL << "add_link, x=" << x_ << " width=" << img_width;
+	PLAIN_LOG << "add_link, x=" << x_ << " width=" << img_width;
 
 	point t_start, t_end;
 
 	setup_text_renderer(curr_item, w_ - x_ - img_width);
 	t_start.x = x_ + get_xy_from_offset(utf8::size(curr_item["text"].str())).x;
 	t_start.y = prev_blk_height_ + txt_height_ - font::get_max_height(font::SIZE_NORMAL);
-	DBG_GUI_RL << "link text start:" << t_start;
+	PLAIN_LOG << "link text start:" << t_start;
```
Comment (CelticMinstrel) [inline src/gui/widgets/rich_label.cpp:217]: These changes seem like they should be reverted?

**Comment by CelticMinstrel** [comment]
Will you be able to rebase this onto gui2_help soon?

**Comment by babaissarkar** [comment]
Accidentally closed, will reopen soon.

**Comment by babaissarkar** [comment]
Moved to #9162.

**Pull request closed without merging**